packages/clay-popover/src/index.tsx

```diff
@@ -43,26 +71,58 @@ interface IProps extends React.HTMLAttributes<HTMLDivElement> {
 const ClayPopover = React.forwardRef<HTMLDivElement, IProps>(
 	(
 		{
+			alignNode,
 			alignPosition = 'bottom',
 			children,
 			className,
 			disableScroll = false,
 			header,
-			show,
 			...otherProps
 		}: IProps,
 		ref
 	) => {
-		return (
+		const [show, setShow] = React.useState(false);
+
+		const alignNodeRef = React.useRef<HTMLElement | null>(null);
+		const popoverRef = React.useRef<HTMLElement | null>(null);
+
+		if (!ref) {
+			ref = popoverRef as React.Ref<HTMLDivElement>;
+		}
+
+		React.useEffect(() => {
```

Comment:
Is the preferred practice to use `React.useSomething` instead of importing `useSomething` from React, as I see examples of both when searching through the source code (mostly of `React.useSomething`)?

Reply:
TBH, it's not a huge issue if you go one way or the other, so long as its consistent for the component or file you are working in. Personally I would opt for `React.*` and I forget where this conversation had happened, but I think we decided to try an opt for `React.*` to avoid confusion with our other custom hooks `use*`.

Reply:
(Not this repo, but...) In liferay-portal we mostly just use `useState` and friends directly, because the number of built-in hooks is small and nobody actually gets confused about them coming from React. Less clutter in the code too, less likely to have to awkwardly wrap lines etc.